### DISCUSSION TITLE
Move item in  a group (after seeing improvements made in #7858)

### DISCUSSION THREAD

**Discussion by jrydmark** [fabricjs/fabric.js#7872] [Q&A] [answered]
Hi all,

We are trying to get a function working for moving an object in a group. Best explained as adding a report band for repeating data and then adding text objects to it. After adding a text object to the repeater object (just a rectangle) it must be possible to move the text object around within that main rectangle object.

Seeing improvements in group handling and I also saw a demo of selecting a grouped object, is it possible to achieve this move functionality for a grouped object?

Any thoughts, hints or ideas on this would be much appreciated. Is it better to create a new main/child object handling type?

Cheers,
Johan

**Comment by ShaMan123** [comment, marked as answer]
Hi Johan,
#7858 has replaced old group with it's predecessor, the new group.
#7859 makes nested selection possible out of the box.
The other group PRs fix a bunch of edge cases so if I were you I would stick at the moment to these 2.
I have updated #7670 you should read it through

**Comment by jrydmark** [comment]
Hi ShaMan123,

Thanks for the hints on where to look. We got it working and is working on updating our Typescript project with it. Now we try to figure out how to stop scaling grouped objects (we want an clip effect instead). 

One step at a time!

Thanks,
Johan

**Reply by ShaMan123** [reply]
#7868

**Reply by ShaMan123** [reply]
http://jsfiddle.net/qxdk4h9L/2/

**Comment by jrydmark** [comment]
Thanks, ShaMan123! Looks like one step further for us in our attempts to build a visual report designer.
